services/tests/endpoint_tests/fixtures.py

```diff
@@ -0,0 +1,138 @@
+import pytest
+from datetime import datetime
+
+from django.contrib.gis.geos import Point, Polygon, MultiPolygon
+from rest_framework.test import APIClient
+
+from services.models import Unit, Department, ServiceNode, Service, UnitServiceDetails
+from munigeo.models import AdministrativeDivisionType, AdministrativeDivision, AdministrativeDivisionGeometry, \
+    Municipality, Address, Street
+
+
+TODAY = datetime.now()
+bbox_0 = MultiPolygon(Polygon.from_bbox((24.916821, 60.163376, 24.960937, 60.185233)), srid=4326)
+bbox_1 = MultiPolygon(Polygon.from_bbox((24.818115, 60.179770, 24.840045, 60.190695)), srid=4326)
+bbox_2 = MultiPolygon(Polygon.from_bbox((24.785500, 60.272642, 25.004797, 60.342920)), srid=4326)
+
```

Comment:
You could add a single dict for all test data, which each fixture then uses to populate the test database, in order to eliminate the need to keep the amount of elements in sync in each fixture.